### ISSUE TITLE
(Improvement/Suggestions) GM Screen/PF2E Support

### ISSUE THREAD

**Issue by Kronos25** [angometry/trinium-chat-buttons#8]
Great module, loving it so far! I don't have an issue to report, but I'd like to leave some suggestions:

- Ability to have more than 12 tabs on the GM screen: 12 is usually more than enough, but I can see some usages that could benefit from more than 12 tabs.
- Support to PF2E automations (similar to some quick access/buttons you've made to Midi QOL)
- GM Screen presets for PF2E

**Comment by angometry** [comment]
I agree about tabs. The next version (which will probably be split into multiple modules) will have the ability to create your own tabs and give them a name instead. You'll be able to hide them in the tab switcher too, to avoid clutter.

About PF2E, I never played it and read the manual too long ago. I wouldn't know how to make presets, but I'll happily add any contribution. Same goes with the buttons, i'll have to take a look at the pf2e system api to see if it can be easily implemented.

I also noticed the ESC issue as soon as I played with the gm screen lol. Not sure whether I want esc to close, a configurable button to toggle it or both, probably just the toggle button.
